src/redis-cli.c

```diff
@@ -9367,6 +9389,7 @@ void bytesToHuman(char *s, size_t size, long long n) {
 static void statMode(void) {
     redisReply *reply;
     long aux, requests = 0;
+    int dbnum = getDatabases();
```

Comment:
how about just aborting the loop when `db%d` of the next index isn't found?
i.e. solely relying on INFO instead of calling an additional CONFIG command.

Reply:
ohh, i see it was already discussed and rejected because redis doesn't print empty databases. :disappointed: